filebeat/beater/crawler.go

```diff
@@ -62,7 +62,7 @@ func newCrawler(
 
 // Start starts the crawler with all inputs
 func (c *crawler) Start(
-	pipeline beat.Pipeline,
+	pipeline beat.PipelineConnector,
```

Comment:
PipelineConnector is the 'smaller' interface. I hope to merge Pipeline and PipelineConnector interfaces in the future, such that Pipeline will not have any extra methods anymore.